src/history/index.js

```diff
@@ -6,25 +6,46 @@ import * as git from './git.js';
 
 export const ROOT_DIRECTORY = path.resolve() + '/data';
 export const RAW_DIRECTORY = `${ROOT_DIRECTORY}/raw`;
+export const SANITIZED_DIRECTORY = `${ROOT_DIRECTORY}/sanitized`;
 
 export async function persistRaw(serviceProviderId, policyType, fileContent) {
-  await storeRaw(serviceProviderId, policyType, fileContent);
-  return await commitRaw(serviceProviderId, policyType);
+  await persist({
+    serviceProviderId, 
+    policyType, 
+    fileContent,
+    isSanitized: false
+  });
 }
 
-export async function storeRaw(serviceProviderId, policyType, fileContent) {
-  const dir = `${RAW_DIRECTORY}/${serviceProviderId}`;
+export async function persistSanitized(serviceProviderId, policyType, fileContent) {
```

Comment:
Isn't this inconsistent with the new `persist({…})` API exposed in this changeset? 🤔